CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 # CHANGELOG
 
+## 1.35.0/2025-05-16
+
+### Added
+* Add Sev0 as a supported incident severity [#2225](https://github.com/DataDog/datadog-api-client-typescript/pull/2225)
+* Share timerestriction object [#2222](https://github.com/DataDog/datadog-api-client-typescript/pull/2222)
```

Comment:
```suggestion
* Share timerestriction object [#2222](https://github.com/DataDog/datadog-api-client-typescript/pull/2222)
```
Please replace "timerestriction" with either "TimeRestriction" to match the object name, or "time restriction" to describe the object.